checkstyle-sonar-plugin/pom.xml

```diff
@@ -330,6 +330,11 @@
                 <branchRate>100</branchRate>
                 <lineRate>78</lineRate>
               </regex>
+              <regex>
+                <pattern>org.sonar.plugins.checkstyle.CheckstyleExecutor</pattern>
+                <branchRate>90</branchRate>
+                <lineRate>94</lineRate>
+              </regex>
```

Comment:
Sorry, this is not accepted.
All new code should have 100% coverage, to prove that it does not have any dead code. 
Yes, we are not ideal now, as we got huge technical dept from previous owner.